java-frontend/src/main/java/org/sonar/java/se/ProgramState.java

```diff
@@ -417,4 +422,9 @@ public SymbolicValue exitValue() {
   public boolean exitingOnRuntimeException() {
     return exitSymbolicValue instanceof SymbolicValue.ExceptionalSymbolicValue && ((SymbolicValue.ExceptionalSymbolicValue) exitSymbolicValue).exceptionType() == null;
   }
+
+  public boolean stackIsEmpty() {
```

Comment:
I removed the method for now, but I think I will reintroduce it once we make also ProgramState.peek to throw an exception